contents/handbook/strategy/overview.md

```diff
@@ -41,30 +41,37 @@ We now add more companies each week than _any_ closed source SaaS rival, and we'
 
 In 2026 we will _go public with $100M ARR._ To achieve this, we will need:
 
-* 1,000 companies building successful products with us, paying $25K each
-* 500 companies building successful products with us, paying $50K each
-* 50 companies building successful products with us, paying $1M each
+1) 1,000 companies building successful products with us, paying $25K each
+2) 500 companies building successful products with us, paying $50K each
+3) 50 companies building successful products with us, paying $1M each
+
+Working on the midmarket first, (1) and (2), is the easiest way to achieve (3) later, from a stronger brand, product and financial position.
```

Comment:
Curious why you've used 'midmarket' instead of 'self serve' here if that's intentional? (As recently we've been talking a lot about building a really strong self-serve engine, rather than focusing on the contract values themselves)

Reply:
happened to discuss with Luke. I now think 'fast growth startups' is the right word:

where:
* fast growth = likely to get lots of users/raise money
* startups = people that don't put us through procurement process

since these are trendy/leading companies that position us as such too, versus ie 'local garage' which could be considered midmarket...

Reply:
Hm yeah I think that works better than 'self serve' as it's more specific. Assuming this is where we land, we should make sure we reiterate this so folks know who we're building/marketing for.